util/retry.go

```diff
@@ -75,6 +75,22 @@ func Retry(op RetriableFunc, attempts int, sleep time.Duration) (bool, error) {
 	return false, errors.New("unable to complete retry operation")
 }
 
+// Retry provides a mechanism to retry an operation with exponential
+// backoff (that uses some jitter,) Specify the maximum number of
+// retry attempts that you want to permit as well as the initial
+// period that you want to sleep between attempts.
+//
+// Retry requires that the starting sleep interval be at least 100
+// milliseconds, and forces this interval if you attempt to use a
+// shorter period.
+//
+// If you specify 0 attempts, Retry will use an attempt value of one.
+func Retry(op RetriableFunc, attempts int, sleep time.Duration) (bool, error) {
```

Comment:
would it be possible to change all uses of Retry to use the context? I think there are less than a dozen.